vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -766,7 +766,7 @@ def process_mmodel_call(self, node, scope):
         mmodel = self.table.get(QName(BaseScope(), mmodel_name))
         if len(mmodel) == 0:
             self.error.append(UninitMModel(mmodel_name.range, mmodel_name.text))
-        else:
+        elif isinstance(mmodel[0], ModularModelCall):

```

Comment:
There's no longer a default fall-through behavior for this 'if'.  Is that OK?

Reply:
In very rare cases the mmodel[0] is not ModularModelCall, with else it will be an error.

Reply:
Is that OK, and is it the expected behavior?  And is the error message you would get reasonable?